examples/tonic-key-value-store/src/main.rs

```diff
@@ -128,8 +153,10 @@ async fn serve_forever(listener: TcpListener) -> Result<(), Box<dyn std::error::
     // Build our database for holding the key/value pairs
     let db = Arc::new(RwLock::new(HashMap::new()));
 
+    let (tx, _rx) = channel(1024);
```

Comment:
minor nit I would prob keep this as `broadcast::channel` or `mpsc::channel` to make it clear which flavor it is.